src/CommonFcComms.hpp

```diff
@@ -166,8 +156,9 @@ void CommonFcComms<T>::updateSensors(const ros::TimerEvent&)
             break;
 
         default:
-            ROS_ERROR("FC_Comms has undefined state.");
+            ROS_FATAL("FC_Comms has undefined state.");
```

Comment:
I think we want these two lines to be just `ROS_ASSERT_MSG(false, "FC_Comms has undefined state.");`.